src/components/ActionList.module.css

```diff
@@ -0,0 +1,32 @@
+.ActionList {
+  list-style: none;
+  white-space: nowrap;
+  text-align: end;
+  min-width: 100%;
+  padding: 0;
+  margin: 0;
+  margin: 5px 0;
+  border-radius: 4px;
+  background-color: #FFFFFF;
+  box-shadow: 0 2px 4px 0 rgba(34,36,38,.12),0 2px 10px 0 rgba(34,36,38,.15);
+  border: 1px solid rgba(34,36,38,.15);
+  z-index: 1;
+
+  position: absolute;
+  top: 100%;
+}
+
+.ActionList > li {
+  font-weight: 500;
+  cursor: pointer;
+  padding: 5px 16px;
+}
+.ActionList > li:hover {
+  background-color: #DEEBFF;
+}
+
+/* CSS to render ActionList dropdown on top of the last ActionButton in Table */
+.LastItemInTable .ActionList {
+  bottom: 100%;
+  top: unset;
+}
```

Comment:
The last ActionButton in the table requires the dropdown to become a dropup, otherwise users would need to scroll a bit more downward to see the dropdown actions.